docs/build.gradle.kts

```diff
@@ -0,0 +1,31 @@
+group = rootProject.group
+version = rootProject.version
+
+plugins {
+    id("ai.kotlin.jvm")
+    alias(libs.plugins.kotlin.serialization)
+    alias(libs.plugins.knit)
+}
+
+repositories {
+    mavenCentral()
+}
```

Comment:
please remove, it is configured for all subprojects: https://github.com/JetBrains/koog/blob/ea30cd9cf3609c5e912f6647e908fe578515f3c0/build.gradle.kts#L70